app/manager-frontend/src/pages/User/list.jsx

```diff
@@ -0,0 +1,5 @@
+export const UsersList = () => {
+  // load Users data from the backend
+  // display Users List (just a placeholder)
+  return <></>;
+};
```

Comment:
Do we need this if it's not doing anything?

Reply:
Yep as Users list action card on the user home page renders this componennt. This component will be filled out in the next PR as a part of Manage Users task (AB#85298), which includes populating users list using backend data.